backend/app/logins.py

```diff
@@ -924,6 +926,7 @@ def get_userinfo(login: LoginStatusDep) -> UserInfo:
     invited_flatpaks = [
         app.app_id
         for _invite, app in models.DirectUploadAppInvite.by_developer(db, user)
+        if not app.archived
```

Comment:
I think we should just default to filtering this on the model. This is also used for the `dev_flatpaks` listing for e.g. and like this it will still show archived repos

Reply:
Not super sure if it's not going to break something else, but sure, we can fix it later